source-code/sdk-js/src/adapter-sveltekit/ast-transformation/transforms/hooks.server.js.test.ts

```diff
@@ -90,19 +96,22 @@ describe("transformHooksServerJs", () => {
 				`)
 			})
 
-			it.skip("function keyword", () => {
-				const code = transformHooksServerJs({} as TransformConfig, dedent`
+			it("function keyword", () => {
+				const code = transformHooksServerJs(
+					{} as TransformConfig,
+					dedent`
 					export function handle({ event, resolve }) {
 						console.log('TADAA!')
 						return resolve(event)
 					}
-				`)
+				`,
+				)
 
 				expect(code).toMatchInlineSnapshot(`
 					"import { initHandleWrapper } from \\"@inlang/sdk-js/adapter-sveltekit/server\\";
 					export const handle = initHandleWrapper({
 					  getLanguage: () => undefined
-					}).wrap(function handle({ event, resolve }) {
+					}).wrap(({ event, resolve }) => {
```

Comment:
please add a TODO to the code. We should not convert a function to an arrow function in the future as it might break something